blockchain/blockchain.go

```diff
@@ -53,7 +66,16 @@ func GetValidations(t string, params common.Params) []int {
 }
 
 func CreateSubscription(sub store.Subscription, params common.Params) store.Subscription {
+	addresses := params.Addresses
+	if len(params.Addresses) == 0 && params.Address != "" {
+		addresses = []string{params.Address}
+	}
```

Comment:
Helper to select a single address if possible

Reply:
We originally started by just having `Addresses`, but on the CL side it makes more sense to only define a single address: `Address`. Having support for both fields is more for backwards compatibility. Ideally, for more safety, I'd prefer users use the single address field.